### PR TITLE
[code] show open in desktop notificaiton

### LINKED ISSUE #5732: [code] show notification to open in VS Code Desktop
@svenefftinge Should we show it once? Or each time and with additional action `Don't show again`? The second approach could be useful as well right now since you always end up in VS Code Web and can get into VS Code Desktop with one click.

### PR DESCRIPTION
**Pull request by akosyakov**
## Description

Show the notification to open in VS Code Desktop on page load.

## Related Issue(s)

Fixes #5732

## How to test

- Switch to latest VS Code.
- Start a workspace.
- You should see a notification, press it, you should open in VS Code Desktop.
- Reload the page, select don't show again, then reload again and test that there is notification anymore.

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **akosyakov** after the PR has been reviewed.

Associated issue: *#5732*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by corneliusludmann — APPROVED** [review]
- commit https://github.com/gitpod-io/openvscode-server/commit/49fa70822b4f1bbb68f29dd3dc1008840ed7128c looks good
- works as advertised

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 25c814894104868a1b91766a861c5e1ef4c5b3dd</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5736#pullrequestreview-757307758" title="Approved">corneliusludmann</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **akosyakov** after the PR has been reviewed.

Associated issue: *#5732*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
@akosyakov: you cannot LGTM your own PR.

<details>

In response to [this](https://github.com/gitpod-io/gitpod/pull/5736#issuecomment-921693468):

>/lgtm


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5736#issuecomment-921693468" title="LGTM">akosyakov</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5736#pullrequestreview-757307758" title="Approved">corneliusludmann</a>*

Associated issue: *#5732*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [akosyakov]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5736#issuecomment-921693735" title="Approved">akosyakov</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5736#pullrequestreview-757307758" title="Approved">corneliusludmann</a>*

Associated issue: *#5732*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [akosyakov]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
